script/update_libgit2

```diff
@@ -5,18 +5,19 @@ set -e
 # augment path to help it find cmake installed in /usr/local/bin,
 # e.g. via brew. Xcode's Run Script phase doesn't seem to honor
 # ~/.MacOSX/environment.plist
-PATH="/usr/local/bin:$PATH"
+if [[ `uname -m` == 'arm64' ]]; then
+  echo "** Running on a Apple Silicon M1"
+  PATH="/usr/homebrew/bin:$PATH"
```

Comment:
IIRC it's `/opt/homebrew/bin` now by default, per https://brew.sh/ samples and:

```
❯ which cmake
/opt/homebrew/bin/cmake
```

Reply:
@DivineDominion  done